traffic_ops/app/lib/API/User.pm

```diff
@@ -477,6 +506,13 @@ sub update_current {
 		return $self->alert( "Profile cannot be updated because '" . $user->{username} . "' is logged in as LDAP." );
 	}
 
+	if ( defined( $user->{"tenantId"} ) ) {
```

Comment:
Can you make it so you can update your current user....but only with an approved tenant?